services/app/src/components/gameFooter/chess-pieces.css

```diff
@@ -48,13 +48,40 @@
 .skip-btn {
   color: #3C64B1;
   background-color: transparent;
+  border: 1px solid black;
 }
 
 .skip-btn:disabled {
   color: rgba(60, 100, 177, 0.24);
   cursor: not-allowed;
 }
 
+.explText{
```

Comment:
When working on `Tooltip` component, I suggest you look into this, to give you a basic idea https://www.w3schools.com/howto/howto_css_tooltip.asp